aws_fuzzy_finder/main.py

```diff
@@ -77,6 +86,24 @@ def entrypoint(use_private_ip, key_path, user):
         lib_path
     )
 
+    ssh_command = ENV_SSH_COMMAND_TEMPLATE.format(
+        user=ENV_SSH_USER or user,
+        key=ENV_KEY_PATH or key_path,
+        host=choice(fuzzysearch_bash_command),
+    )
+
+    if vpc:
+        ssh_command += " -t " + ENV_SSH_COMMAND_TEMPLATE.format(
+            user=ENV_VPC_SSH_USER or vpc_user,
+            key=ENV_VPC_KEY_PATH or vpc_key_path,
+            host=choice(fuzzysearch_bash_command),
+        )
+
+    print(ssh_command)
+    subprocess.call(ssh_command, shell=True, executable='/bin/bash')
+
+
+def choice(fuzzysearch_bash_command):
```

Comment:
I love splitting this to a separate function, and reusing it :+1: It lets us use the prompt twice, and is very convenient. :+1: